python-lib/youtube_client.py

```diff
@@ -351,6 +344,17 @@ def has_data_to_process(self):
         else:
             return self.has_next_page()
 
+    def set_item_number_limit(self, max_item_number):
+        self.max_item_number = max_item_number
+
+    def has_remaining_data(self, nb_items):
```

Comment:
This function is more a control on the fact that you can ask weither or not there is some remaining data